### PR TITLE
Sourav/benchmarks

### PR DESCRIPTION
**Pull request by souravDgraph**
The Yahoo! Cloud Serving Benchmark (YCSB) is often used to compare relative performance of NoSQL database management systems. It has different load types:
 1. YCSB A (50% reads, 50% updates, zipf key distribution) 
2.  YCSB B (95 reads, 5% updates, zipf key distribution) 
3.  YCSB C (100% reads, zipf key distribution) 
4.  YCSB D (95% reads, 5% updates, uniform key distribution) 
5.  YCSB E (95% scans, 5% updates, zipf key distribution) 
6. Insert-only (100% inserts, zipf key distribution) 
The current run can be seen detailed in : http://dgin-delta.corp.dgraph.io:4040/main.html?max=local

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1659)

### CODE REVIEW

**Review by jarifibrahim — CHANGES_REQUESTED** [review]
There are too many files. Let's remove all the unnecessary files.

**Review by NamanJain8 — COMMENTED** [review]
I see a few empty files. Are they needed? 
Also, is the part of the code ported from somewhere?

**Comment by souravDgraph** [comment]
@NamanJain8 - Most of the code is integrated from https://github.com/cockroachdb/pebble/tree/master/cmd/pebble
The empty files are majorly the prof files generated when we run the benchmark. Since the content will be dependent on the system in which we run, so I have removed the same.

File: go.mod [hunk 1]
```diff
@@ -7,19 +7,25 @@ go 1.12
...
+	github.com/cockroachdb/pebble v0.0.0-20210120202502-6110b03a8a85
+	github.com/cockroachdb/redact v1.0.8
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd
+	github.com/dgraph-io/badger/v2 v2.2007.2
 	github.com/dgraph-io/ristretto v0.0.4-0.20210205182321-f8e4908e34d1
 	github.com/dustin/go-humanize v1.0.0
-	github.com/gogo/protobuf v1.3.2
-	github.com/golang/protobuf v1.3.1
-	github.com/golang/snappy v0.0.1
+	github.com/gogo/protobuf v1.3.1
```
Comment (NamanJain8) [inline go.mod:17]: Undo this version change.

File: go.mod [hunk 2]
```diff
@@ -7,19 +7,25 @@ go 1.12
...
+	github.com/golang/protobuf v1.4.2
+	github.com/golang/snappy v0.0.2-0.20190904063534-ff6b7dc882cf
```
Comment (NamanJain8) [inline go.mod:19]: We should not use unreleased version. Maybe undo this as well.

File: go.sum [hunk 3]
```diff
@@ -17,16 +46,48 @@ github.com/creack/pty v1.1.9/go.mod h1:oKZEueFk5CKHvIhNR5MUki03XCEU+Q6VDXinZuGJ3
...
+github.com/go-errors/errors v1.0.1/go.mod h1:f4zRHt4oKfwPJE5k8C9vpYG+aDHdBFUsgrm6/TyX73Q=
+github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200222043503-6f7a984d4dc4/go.mod h1:tQ2UAYgL5IevRw8kRxooKSPJfGvJ9fJQFa0TUsXzTg8=
+github.com/go-martini/martini v0.0.0-20170121215854-22fa46961aab/go.mod h1:/P9AEU963A2AYjv4d1V5eVL1CQbEJq6aCNHDDjibzu8=
+github.com/gobwas/httphead v0.0.0-20180130184737-2c6c146eadee/go.mod h1:L0fX3K22YWvt/FAX9NnzrNzcI4wNYi9Yku4O0LKYflo=
+github.com/gobwas/pool v0.2.0/go.mod h1:q8bcK0KcYlCgd9e7WYLm9LpyS+YeLd8JVDW6WezmKEw=
+github.com/gobwas/ws v1.0.2/go.mod h1:szmBTxLgaFppYjEmNtny/v3w89xOydFnnZMcgRRu/EM=
+github.com/gogo/googleapis v0.0.0-20180223154316-0cd9801be74a/go.mod h1:gf4bu3Q80BeJ6H1S1vYPm8/ELATdvryBaNFGgqEef3s=
+github.com/gogo/protobuf v1.2.0/go.mod h1:r8qH/GZQm5c6nD/R0oafs1akxWv10x8SbQlK7atdtwQ=
+github.com/gogo/protobuf v1.3.1 h1:DqDEcV5aeaTmdFBePNpYsp3FlcVH/2ISVVM9Qf8PSls=
+github.com/gogo/protobuf v1.3.1/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
```
Comment (NamanJain8) [inline go.sum:87]: Run `go mod tidy`

File: internal/manifest/testdata/version_edit_apply [hunk 4]
```diff
@@ -0,0 +1,169 @@
+apply
```
Comment (NamanJain8) [inline internal/manifest/testdata/version_edit_apply:1]: I hope we need this file. And this is not the generated one.

**Review by NamanJain8 — COMMENTED** [review]
Some minor comments. Generated files should not be in the codebase. Are they needed for code to run? Or they are generated when the code is run?

Also, some README would be useful for reference on how to run this benchmark etc.

**Pull request closed without merging**